gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/datafetcher/EntityDataServiceEntityFetcher.java

```diff
@@ -182,17 +183,8 @@ public EntityFetcherResponse getTimeAggregatedMetrics(
   }
 
   @Override
-  public int getTotalEntities(EntitiesRequestContext requestContext, EntitiesRequest entitiesRequest) {
-    EntityFetcherResponse entityFetcherResponse = getEntities(
-        requestContext,
-        EntitiesRequest.newBuilder(entitiesRequest)
-            .clearSelection()
-            .clearTimeAggregation()
-            .clearOrderBy()
-            .clearLimit()
-            .setOffset(0)
-            .build()
-    );
-    return entityFetcherResponse.size();
+  public Set<EntityKey> getTotalEntities(EntitiesRequestContext requestContext, EntitiesRequest entitiesRequest) {
```

Comment:
This method implies that a number is the return value. Can it be renamed to reflect that we are returning a set?

Reply:
in fact, got rid of the method. wasn't needed https://github.com/hypertrace/gateway-service/pull/61#discussion_r558370431